doc/tutorial.rst

```diff
@@ -265,7 +255,7 @@ in a ''dbus'' class property or in its ''docstring''. For example::
       def SomeProperty(self):
         return self._someProperty
 
-      @SomeProperty.setter
+      @signalled(SomeProperty)
```

Comment:
For example, our object is a "CD drive", and our property is "DiskAvailable". User presses the physical eject button, and kernel ejects the drive. Our daemon detects that fact, and emits a signal.